source/agora/consensus/protocol/Nominator.d

```diff
@@ -492,10 +500,10 @@ extern(D):
 
     ***************************************************************************/
 
-    protected TimePoint getExpectedBlockTime () @safe @nogc nothrow pure
+    protected TimePoint getExpectedBlockTime (Height height) @safe @nogc nothrow pure
```

Comment:
Since `height+1` behavior also changed; DDoc for argument would be good.

Reply:
Yup, the change is quite orthogonal to stats. Actually, perhaps moving this functionality to `Ledger` (next to `expectedHeight`) would make sense ?